Added some details to custom menu documentation

MateMahinKhan opened the pull request:
<!-- make sure your title is clear -->
<!-- to mark a task, use [x]. -->
<!-- try not to make breaking changes without discussion first -->
<!-- don't remove this template. else PR might be closed without any discussion -->

This PR contains:

- [ ] bugfix
- [ ] feature
- [ ] refactor
- [ ] tests
- [x] documentation
- [ ] others

### Breaking Changes?

- [ ] yes
- [x] no

<!-- If yes, please describe the breakage. -->

### Checklist

- [ ] commit messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
  - A document related commit is prefixed "docs:"
- [x] docs have been added / updated (for bug fixes / features)

### Describe Your Changes

Added some details for Custom Menu. There wasn't anything missing from the existing documentation but I've added some explicit details.

### Does this PR affects any existing issues?

- [ ] yes
- [x] no

<!--
  if yes mention the issue number and what kind of change it is
  for example: closes #1, fixes #1.
-->

sibiraj-s commented on docs/menu.md:
```diff
@@ -55,7 +55,7 @@ export class AppComponent implements OnInit, OnDestroy {
 ## Custom Menu
 
-The editorView will be available from the init function
+The editorView will be available from the init function. Create `customMenu` as a template variable using the custom-menu component.
```
```suggestion
Create `customMenu` as a template variable using the custom-menu component.
```

init function no longer exists. can you remove this line

sibiraj-s replied:
> Create `customMenu` as a template variable using the custom-menu component.

this somehow feels like the editor is providing a custom menu component to create custom, but it is create a template and pass the ref as a input to the menu component, via `customMenuRef`

sibiraj-s commented on docs/menu.md:
```diff
@@ -1,6 +1,6 @@
 # Menu
 
-Menu is not part of the editor component. Include `ngx-editor-menu` in your HTML manually
+Menu is not part of the editor component. Include `ngx-editor-menu` in your HTML manually. The custom-menu items will be added after the items on the toolbar.
```
```suggestion
Menu is not part of the editor component. Include `ngx-editor-menu` in your HTML manually. The custom-menu will be inserted after the toolbar items.
```

sibiraj-s replied:
and this line can be moved into the `CustomMenu` section. The info seems early.

sibiraj-s reviewed (changes requested):
Thanks for the PR. left some suggestions.

MateMahinKhan commented:
Incorporated your suggestions. 

I changed the line to `Create 'customMenu' as a template variable using the newly created custom-menu component.` I'm not sure adding newly created might fix the confusion.

sibiraj-s reviewed (approved):
Thanks @MateMahinKhan.

sibiraj-s commented:
I made some changes to the custom menu part. I hope it is explanatory. 

Also, please follow the commit guidelines. I will squash merge this for now.
